utils/auth.js

```diff
@@ -2,8 +2,8 @@
 const withAuth = (req, res, next) => {
     // If the user is not logged in, redirect the user to the login page
     // This is directly from the `/gallery/:id` and `/painting/:id` routes
-    if (!req.session.logged_in) {
-      res.status(401).end();
+    if (!req.session.loggedIn) {
+      res.redirect('/login');
```

Comment:
Could you please replace line 6 with this:
      //This condition protect the api routes and front-end routes 
      if(req.originalUrl.indexOf('api') > -1){
         res.status(401).json({
          status: 401,
          message: "Unautherized user not loged in!"
        });
      }else{
        res.redirect('/login');
      }

this way we don't break each others code.

Reply:
that breaks the login